### ISSUE TITLE
add metrics-server to MDE

### ISSUE THREAD

**Issue by bit-herder** [revelrylabs/moondog-engine#4]
Background:
- we can't run like `kubectl top pods` or `kubectl top nodes` without metrics-server

A/C:
- [ ] we have added metrics-server template to MDE

**Comment by bit-herder** [comment]
we need to do a bit of work internally to output the correct cert to use here before we get this merged up. Kubadm apparently does not use the same cert. 

see issue here : https://github.com/kubernetes/kubeadm/issues/1223

and here: https://github.com/kubernetes-sigs/metrics-server/blob/master/FAQ.md#how-to-run-metrics-server-securely

**Comment by jwietelmann** [comment]
Here's a good example of what you'll need to do: https://github.com/revelrylabs/moondog-engine/blob/master/templates/gangway.yaml

Notes:

* I should have used a ConfigMap instead of a Secret because there's no need to make `.crt` files secret
* The `metrics-server` chart provides config sections for `extraVolumes` and `extraVolumeMounts` just like the `gangway` chart does: https://github.com/helm/charts/tree/master/stable/metrics-server#configuration
* tl;dr - Make a ConfigMap to hold the cert, use the chart config to mount it at some path or another, then use `args` to point metrics-server at that path.

The part I feel like I personally still don't know is where to look for the right cert file to use.

**Comment by jwietelmann** [comment]
Starting to think that, given how few people set this up securely or even have their kubelets configred such that a secure install is possible, including this in MDE would add prerequisite steps that are unrealistic for the vast majority of users. Maybe if revelry really wants this, we should just drop our own HelmRelease into our own flux repos instead of having MDE do it.

**Comment by iamjoshfrank** [comment]
Based on what I've been told, and following the convo re: possible solutions, I agree. We can/should drop this from MDE and let it be a manual install if needed until the security issue(s) are resolved.

Let's assign @bit-herder to close this out or make an argument for including it, but with some changes.

**Comment by bit-herder** [comment]
Thats fine for now. Closing.
